public/index.php

```diff
@@ -4,6 +4,26 @@
  * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
+if (ini_get('display_errors')) {
+    /*
+     * PHP by default returns HTTP 200 even when exceptions are thrown.
+     *
+     * Now by default return HTTP 500 so any API client can trust in the HTTP response code.
+     */
+    http_response_code(500);
```

Comment:
I don't know but for me the thing of XDebug is inside of the scope of this PR.

My original issue was run API tests and tests fails for json unencode issues without more information.

Now with a 500 code I can intercept the error first and print more useful messages.